AdminWebsite/AdminWebsite/ClientApp/src/app/app.component.ts

```diff
@@ -48,11 +53,32 @@ export class AppComponent implements OnInit {
     ngOnInit() {
         this.checkBrowser();
 
+        this.configService
+            .getClientSettings()
+            .pipe(first())
+            .subscribe({
+                next: clientSettings => {
+                    this.dynatraceService.addDynatraceScript(clientSettings.dynatrace_rum_link);
+                }
+            });
+
         this.securityService.checkAuthMultiple().subscribe(response => {
             const user = response.find(x => x.configId === this.securityService.currentIdpConfigId && x.isAuthenticated);
+
             if (user) {
                 this.loggedIn = true;
                 this.username = user.userData?.preferred_username?.toLowerCase();
+
+                /* The line
+                `this.dynatraceService.addUserIdentifyScript(userData?.preferred_username?.toLowerCase());`
+                is calling a method `addUserIdentifyScript` from the `dynatraceService`
+                service. This method is used to identify the user in Dynatrace
+                monitoring by passing the user's preferred username in lowercase as a
+                parameter.*/
+                // TODO: uncomment the below line to add this identification
+                // To be implemented behind the cookie banner VIH-5622
+
```

Comment:
It's not commented out in the admin-web version? Probably needed I'm assuming as this isn't public facing, im assuming. In which case you can delete the comment

Reply:
comment removed